src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -1180,5 +1282,26 @@ private void OnFrameRendered(IntPtr data)
         {
             frameRenderedEventHandler?.Invoke(this, new EventArgs());
         }
+
+        private void OnCertificateConfirmed(IntPtr data, IntPtr certificate)
+        {
+            WebCertificate webCertificate = new WebCertificate(certificate, false);
```

Comment:
How about changing this?
It does not create unnecessary objects.
```csharp
certificateConfirmedEventHandler?.Invoke(this, new WebViewCertificateReceivedEventArgs(new WebCertificate(certificate, false)));
```

Reply:
Done